api/cluster.capnp

```diff
@@ -6,6 +6,29 @@ $Go.package("cluster");
 $Go.import("github.com/wetware/ww/internal/api/cluster");
 
 
+interface Anchor {
+    ls @0 (path :List(Text), handler :Handler) -> ();
+    walk @1 (path :List(Text)) -> (anchor :Anchor);
+
+    interface Handler {
```

Comment:
I forgot to bring this up during our calls, but I think we only really need streaming responses for the root anchor (_i.e._ the one that returns host anchors).  By grouping related anchors under a common parent, users can ensure a given anchor doesn't have an excessive number of children.  This is analogous to how you keep your filesystem tidy:  when a directory has too many files, you group these files into subdirectories.

In this manner, the root anchor is actually the exception because you cannot group hosts under different parent anchors.  This is why we have to be careful about streaming host anchors back to the client.

In sum, I think it would be better to only stream results from the root anchor (_i.e._ `cluster.Node`).  Everything else should simply return `List(AnchorElement)`.  This will greatly simplify implementation, and provide a moderate improvement in network performance.